### DISCUSSION TITLE
Custom scalar coercing exception handler

### DISCUSSION THREAD

**Discussion by Meemaw** [Netflix/dgs-framework#1913] [General]
Hi,

I have a question about custom scalars in dgs. When throwing `CoercingParseLiteralException` or similar in the `Coercing`, is there a way to disable the errors logs that are emitted?

Usually those errors are not server/internal errors, but more likely a user input error. This is a easy attack vector to spam error logs in your GraphQL server.

**Comment by srinivasankavitha** [comment]
Have you tried just disabling the specific class logging for slf4j? Something like this should work to turn off that particular error:
```
logging.level.com.test.MyClass=WARN
```
